user_docs/en/userGuide.t2t

```diff
@@ -940,8 +940,7 @@ There are many languages, however the default option is "User Default, Windows".
 This option tells NVDA to use the language that Windows is currently set to.
 
 Please note that NVDA must be restarted when changing the language.
-NVDA will ask you if you wish to restart if you do change the selection and save your changes.
-Press OK, and NVDA will restart.
+When the confirmation dialog appears, select "restart now" or "restart later" if you wish to use the new language now or at a later time, respectively.

```

Comment:
I think this should mention 'If "restart later" is selected, the configuration must be saved (either manually or using the save on exit functionality)'

@Qchristensen do you have an opinion on the wording of this?